Updated readme file

amuza2 opened the pull request:
added the full name of CCP and RIP,
then added the result of each method in a comment next to it.

amuza2 committed:
updated the readme file

amuza2 committed:
updated readme

amuza2 committed:
updated readme

amuza2 committed:
updated readme

amuza2 committed:
updated readme

mouh2020 reviewed (changes requested):
Please update the return value of get_rip_cle() to 06.I have resolved the problem where the clé had only one digit and now it should have two digits by adding a leading zero if needed.

amuza2 committed:
Update README.md

updated 6 to 06

amuza2 commented:
> Please update the return value of get_rip_cle() to 06.I have resolved the problem where the clé had only one digit and now it should have two digits by adding a leading zero if needed.

Done

amuza2 committed:
simplified the CCP class

amuza2 committed:
made the calculate_rip method private

amuza2 commented:
I added two commits
1- I wanted to simplify the code for better readability, I extracted the RIP calculation into a separate method calculate_rip to avoid code duplication. This should make it easier for a programmer to read and understand the code.
2- I made the calculate_rip method private to prevent users from directly accessing it.

amuza2 committed:
Merge branch 'master' into update-01
